package.json

```diff
@@ -31,7 +31,7 @@
     "standard": "^14.3.4"
   },
   "peerDependencies": {
-    "react": "^16.12.0"
+    "react": "^16.12.0 || ^17.0.2 || ^18.2.0"
```

Comment:
I'm never seen dependencies versioned like this with the ||.  nifty!

Reply:
It's neat. I think just doing `>=16.12.0` might also work, but who knows when a new react version will break it